cmd/root.go

```diff
@@ -39,6 +39,7 @@ func init() {
 	viper.SetConfigName("config")
 	viper.SetConfigType("yaml")
 	viper.AddConfigPath(defaultConfigPath)
+	viper.AddConfigPath("$XDG_CONFIG_HOME/zube")
```

Comment:
Do not use hardcoded path separators, use https://pkg.go.dev/path/filepath#Join instead

Reply:
Oh, so that's the OS-agnostic way, we have to keep track of that ourselves. Noted 👍